src/prefect/utilities/callables.py

```diff
@@ -149,10 +149,18 @@ class ModelConfig:
         else:
             name = param.name
 
+        # If an override is provided, use that, otherwise use the default value
+        # from the flow definition
+        parameter_default = None
```

Comment:
We should still probably represent `empty` with `...` instead of `None` — the meanings are distinct.